packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -210,6 +210,16 @@ export const makeZCFZygote = async (
   });
   const handleOfferObj = makeHandleOfferObj(taker);
 
+  /**
+   * @type {() => Promise< {
+   * buildRootObject: any,
+   * start: undefined,
+   * meta: undefined,
+   * } | {
+   * buildRootObject: undefined,
+   * start: ContractStartFn,
+   * meta?: ContractMeta,
+}>} */
```

Comment:
The formatting looks broken here. Is github confused?

Reply:
That is janky. I must have messed it up.

prettier-plugin-jsdoc would format it as,
```
  /**
   * @type {() => Promise<
   *   | {
   *       buildRootObject: any;
   *       start: undefined;
   *       meta: undefined;
   *     }
   *   | {
   *       buildRootObject: undefined;
   *       start: ContractStartFn;
   *       meta?: ContractMeta;
   *     }
   * >}
   */
```
Since that automated our whitespace, I'll address this when we opt packages/zoe into pretty jsdoc

Reply:
I started noticing that VS code is not auto formatting for some packages (i.e. cosmic-swingset). It does for SwingSet. Maybe I started noticing this in cosmic-swingset because I just now made some changes in it.

Reply:
Please ignore my comment. I was missing settings file for cosmic-swingset in my project.